### PR TITLE
moveit ik

### PR DESCRIPTION
**Pull request by SammyRamone**
This PR replaces the old way to call the IK with a much faster one.
This is fast enough to replace the need of an analytic IK.

## Proposed changes
Call IK via normal MoveIt! interface rather than via ik_options.

## Related issues
#109 

## Necessary checks
- [x] Update package version
- [x] Run linters
- [x] Run `catkin build`
- [x] Write documentation
- [x] Test on your machine
- [x] Test on the robot

### CODE REVIEW

**Comment by SammyRamone** [comment]
was tested on robot but not after code clean up. needs a final test

File: bitbots_quintic_walk/config/walking_wolfgang_viz.yaml [hunk 1]
```diff
@@ -74,7 +74,7 @@ walking:
     engine_freq: 100.0
 
     # parameters for bioIK
-    bio_ik_time: 0.01
```
Comment (timonegk) [inline bitbots_quintic_walk/config/walking_wolfgang_viz.yaml]: This parameter should probable be renamed to ik_timeout here and in the code too.

**Comment by timonegk** [comment]
Does this replace #109?

File: bitbots_quintic_walk/config/walking_wolfgang_viz.yaml (hunk 1 above)
Comment (SammyRamone) [reply bitbots_quintic_walk/config/walking_wolfgang_viz.yaml]: correct

**Comment by SammyRamone** [comment]
> Does this replace #109?

yes, I closed the other PR

**Pull request merged**